src/components/RegisterWallet/PolicyRegistrationsTable.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useMemo } from "react";
+import PropTypes from "prop-types";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Typography,
+} from "@material-ui/core";
+import { useSelector } from "react-redux";
+
+const PolicyRegistrationTable = ({ hmacs }) => {
+  const extendedPublicKeyImporters = useSelector(
+    (state) => state.quorum.extendedPublicKeyImporters
+  );
+
+  const hmacsWithName = useMemo(() => {
+    return Object.values(extendedPublicKeyImporters)
```

Comment:
this would be a great candidate for creating a new selector for grabbing just the values using reselect lib.